.github/workflows/website.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          repository: 'tridactyl/site'
+            # submodules: 'recursive' # we shouldn't need this, public should be generated each time
```

Comment:
It should be safe to delete the submodule, right? I think we set the site up before GitHub actions existed. The submodule was a hacky way around it since the page itself needed to be served from the tridactyl.github.io repo

Reply:
I don't recall exactly what the problem was, but cloning without submodules made the build fail in an obscure way (something about hugo not being able to find some files?). I believe some of the files required in order to build the site have been comitted in the submodule instead of the tridactyl/site repo itself.